src/test/java/uk/gov/companieshouse/overseasentitiesapi/model/DtoModelChangeTest.java

```diff
@@ -198,9 +198,7 @@ private boolean isAListOrMapOfStrings(Object dto) {
 
     private boolean ignoreForNow(String fieldName) {
         // TODO Set values for these fields in the 'overseas_entity_v_3_1.json'
-        return fieldName.equals("locality") || fieldName.equals("poBox") || fieldName.equals("careOf") || fieldName.equals("town")
-                || fieldName.equals("chipsReference")
-                || fieldName.equals("secondNationality")
+        return  fieldName.equals("careOf") || fieldName.equals("town")
```

Comment:
Just wondering why you didn't take 'care of' `careOf` and `town` when you dealt with the other two fields?

Reply:
Now clear why.

Reply:
yes, just time limited, only managed those so far, will do some more in another PR